week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,31 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp() {
+    this.volume += 100; // calling this function should pour your bottle full (volume = 100);
   },
-  pour: function () {
-    // calling this function should increase your bottle volume by 10 unit;
+  pour() {
+    if (this.volume <= 90) {
+      return (this.volume += 10); // calling this function should increase your bottle volume by 10 unit;
+    }
   },
-  drink: function() {
-    // calling this function should decrease your bottle volume by 10 unit;
+  drink() {
+    if (this.volume >= 10) {
+      return (this.volume -= 10); // calling this function should decrease your bottle volume by 10 unit;

```

Comment:
while the >= is okey.do
what about using a more clear comparison 
like 
this.volume>0

Reply:
I am not sure if I have done something wrong but If I move '=' here and use only > the test fails.

Reply:
Your work is not wrong but depends on the way you wrote the condition
let's say I want to write it in a different way
```drink(){
this.volume-=10
if(this.volume<0){
this.volume=0
}
```